data/brands/office/financial.json

```diff
@@ -66,6 +66,16 @@
         "brand:wikipedia": "en:OneMain Financial",
         "name": "OneMain Financial",
         "office": "financial"
+      },
+      {
+      "displayName": "Ymanci",
+      "locationSet": {"include": ["fr"]},
+      "tags": {
+        "brand": "Ymanci",
+        "name": "Ymanci",
+        "operator": "Premista",
+        "office": "financial",
+        "note": "Three branches: Assurances, Crédit immobilier and Regroupement de crédits"
```

Comment:
```suggestion
```

Reply:
In this case I wanted the note to be adressed to NSI users in case of further development of the operator, not towards OSM users. I thought it fitted the "brand's status" note category.

Reply:
so you don‘t need to put it here
if it don’t for OSM mapper but for NSI dev，it should outer `tags`